src/components/HomePage/URLSearchParamsParser.js

```diff
@@ -7,12 +7,21 @@ import { useSelector } from "react-redux";
 import useOffersSearcher from "./SearchResultsArea/SearchResultsWidget/useOffersSearcher";
 import { SearchResultsConstants } from "./SearchResultsArea/SearchResultsWidget/SearchResultsUtils";
 
+// offerSearch params that should NOT trigger an auto-submit
+const invalidParams = ["error", "offers", "loading", "filterJobDuration"];
+
 const URLSearchParamsParser = ({ showSearchResults }) => {
 
     const location = useLocation();
 
     const offerSearch = useSelector((state) => state.offerSearch);
-    const validQueryParams = Object.keys(offerSearch);
+
+    const validQueryParams = Object.keys(offerSearch).filter((field) => !(invalidParams.includes(field)));
+    if (validQueryParams.includes("jobDuration")) {
+        validQueryParams.splice(validQueryParams.indexOf("jobDuration"), 1);
+        validQueryParams.push("jobMaxDuration");
+        validQueryParams.push("jobMinDuration");
+    }
```

Comment:
```suggestion
    if (validQueryParams.includes("jobDuration")) {
        validQueryParams.splice(validQueryParams.indexOf("jobDuration"), 1, "jobMaxDuration", "jobMinDuration");
    }
```
This should be the same. What do you think?